### PR TITLE
docs: add garbage-collection.md for gc

### PR DESCRIPTION
**Pull request by Desiki-high**
Signed-off-by: Yadong Ding <ding_yadong@foxmail.com>

### CODE REVIEW

File: docs/garbage-collection.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Garbage Collection can be triggered in two ways. Firstly, gc will be triggered after each
+task completed. Secondly, gc has scheduled tasks every hourly since dameon started. GC clears
+the blobs until local storage size below 80% of the threshold, scheduled task is 40%.
+
+## Garbage Collection Configuration
+
+| Configuration | Description |
+|---|---|
+| `gcpolicy.threshold` | size threshold that triggers GC, the `gc` will start reclaim blobs if exceeds the size. |
```
Comment (imeoer) [inline docs/garbage-collection.md:29]: In what's unit here?

File: docs/garbage-collection.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+new blob and delete leases of the blobs which should be cleared away.
+The `leaseCache` will sync with the leases which are persisted in boltdb and provide
+the lease that should be cleared in LFRU(LFU first, LRU second) order. The `acceld` will
+init `leaseCache` when daemon started.
+
+## Garbage Collection Policy
+
+Garbage Collection can be triggered in two ways. Firstly, gc will be triggered after each
+task completed. Secondly, gc has scheduled tasks every hourly since dameon started. GC clears
+the blobs until local storage size below 80% of the threshold, scheduled task is 40%.
```
Comment (imeoer) [inline docs/garbage-collection.md:23]: What's the meaning of `scheduled task is 40%` here?

File: docs/garbage-collection.md (hunk 1 above)
Comment (Desiki-high) [reply docs/garbage-collection.md:29]: I can add a example to introduce

File: docs/garbage-collection.md (hunk 2 above)
Comment (Desiki-high) [reply docs/garbage-collection.md:23]: ```scheduled gc task``` will clear the blobs until the local storage size is below 40% of the threshold, which means the half of ```convert gc task```.

**Review by imeoer — APPROVED** [review]
LGTM!

**Pull request merged**